Content.Shared/Singularity/Components/SharedGravityWellComponent.cs

```diff
@@ -0,0 +1,7 @@
+namespace Content.Shared.Singularity.Components;
+
+/// <summary>
+/// A component that makes the associated entity attract/repel/vorticerate other entities within some distance.
+/// </summary>
+public abstract class SharedGravityWellComponent : Component
+{}
```

Comment:
Again, just make this a single component in shared,

Reply:
Gravity well has no behavior on the client. I moved it to the server instead.